### PR TITLE
Linking to javadoc inner classes

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

**Comment by ennru** [comment]
See https://github.com/lightbend/sbt-paradox-apidoc/issues/86

**Comment by ennru** [comment]
Any reason why you didn't add tests to the existing `JavadocDirectiveSpec` in `tests`?

**Comment by raboof** [comment]
> Any reason why you didn't add tests to the existing JavadocDirectiveSpec in tests?

Oops - because I didn't need a testkit and didn't realize there was one. I'll move it.

File: core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala [hunk 1]
```diff
@@ -333,6 +334,16 @@ object JavadocDirective {
   val jdkDependentLinkStyle = if (sys.props.get("java.specification.version").exists(_.startsWith("1."))) LinkStyleFrames else LinkStyleDirect
 
   final val JavadocLinkStyleProperty = raw"""javadoc\.(.*).link_style""".r
+
+  private[markdown] def url(link: String, baseUrl: Url, linkStyle: LinkStyle): Url = {
+    val url = Url(link).base
+    val path = url.getPath.replaceAll("(\\b[a-z]+)\\.", "$1/") + ".html"
```
Comment (ennru) [inline core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala]: This relies on the naming convention of packages being all-lowercase (which is rarely broken).

**Pull request merged**